pkgs/development/libraries/gnutls/generic.nix

```diff
@@ -16,7 +16,6 @@ stdenv.mkDerivation {
   outputs = [ "out" "man" ];
 
   configureFlags =
-    # FIXME: perhaps use $SSL_CERT_FILE instead
     lib.optional stdenv.isLinux "--with-default-trust-store-file=/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Because GnuTLS does not depend on OpenSSL, it is another implementation.
That's why we need to agree on a common default store.